src/main/java/mike/Parser.java

```diff
@@ -5,11 +5,29 @@
 import mike.task.Task;
 import mike.task.ToDo;
 
+/**
+ * Parses input from the user
+ */
 public class Parser {
+
+    /**
+     * Returns a Command with the enum type by parsing the input
+     *
+     * @param input String object representing the user's input
+     * @return a Command of the enum type
+     */
     public static Command parseCommand(String input) {
         return Command.valueOf(input.toUpperCase());
     }
 
+    /**
+     * Returns a Task Object by parsing the description depending on the Command
+     *
+     * @param command an object of the Command type
+     * @param input the input of the user
+     * @return the Task corresponding to the user's command
+     * @throws ParseException if parsing fails
+     */
     public static Task parseDescription(Command command, String input) throws ParseException {
```

Comment:
I think from first glance, it could be useful if you described in greater detail what you mean by "an object of the Command type", but overall pretty self-explanatory method names. 👍